pkg/ctl/delete/cluster.go

```diff
@@ -103,11 +104,23 @@ func doDeleteCluster(rc *cmdutils.ResourceCmd) error {
 	}
 
 	{
+
+		logger.Info("cleaning up LoadBalancer services")
+		if err := ctl.GetCredentials(cfg); err != nil {
+			return err
+		}
+		client, err := ctl.NewClient(cfg, false)
+		if err != nil {
+			return err
+		}
+		elb.CleanupLoadBalancers(ctl.Provider.ELB(), ctl.Provider.ELBV2(), client)
+
 		tasks, err := stackManager.NewTasksToDeleteClusterWithNodeGroups(rc.Wait, func(errs chan error, _ string) error {
 			logger.Info("trying to cleanup dangling network interfaces")
 			if err := ctl.GetClusterVPC(cfg); err != nil {
 				return errors.Wrapf(err, "getting VPC configuration for cluster %q", cfg.Metadata.Name)
 			}
+
 			go func() {
 				errs <- vpc.CleanupNetworkInterfaces(ctl.Provider.EC2(), cfg)
```

Comment:
This ENI callback needs some refactoring, will need to be addressed as part of #536.